Modules/Core/Support/TemplateHelper.php

```
@@ -0,0 +1,187 @@
+    /**
+     * Parse a template by predefined template tags.
+     *
+     * @param $object
+     * @param $body
+     * @param $model_id
+     *
+     * @return mixed
+     */
+    public static function parse_template($object, $body)
+    {
+        if (preg_match_all('/{{{([^{|}]*)}}}/', $body, $template_vars)) {
```

PHPMD (2.15.0)

The variable $template_vars is not named in camelCase. (undefined)

(CamelCaseVariableName)